analyses/metacells/scripts/run-seacells.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+
+"""
+Seacells Analysis Script
+Joshua Shapiro
+2024-07-18
+
+This script runs the SEACells algorithm on a dataset.
+"""
+
+# Load modules
+#
+# Load required Python modules at the top of your script
+# We have included the standard `pathlib` module and the `session_info` module
+# that we will be using at the bottom of this notebook to record the versions of
+# the modules used in this analysis.
+#
+# Do not install modules here; only load them with `import` statements.
+# Avoid renaming modules with `as` statements, unless you are performing a
+# standard renaming (e.g., `import pandas as pd`).
+
+import argparse
+import contextlib
+import pathlib
+import pickle
+import sys
+
+import anndata
+import numpy as np
+import scanpy as sc
+import SEACells
+import session_info
+
+
+def convert_adata(adata: anndata.AnnData) -> anndata.AnnData:
+    """
+    Convert an ScPCA AnnData object to the formatting expected by scanpy and SEACells.
+    """
+    # put the highly variable genes in a column of adata.var
+    adata.var["highly_variable"] = adata.var.gene_ids.isin(
+        adata.uns["highly_variable_genes"]
+    )
+
+    # recompute principal components and umap (adds additional metadata)
+    sc.tl.pca(adata, n_comps=50, mask_var="highly_variable")
```

Comment:
any specific reason for 50?

Reply:
Intent here is to keep it the same as the values already present, but I will make that explicit and check.